<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="AllAccountMatching" time="73" tests="1" failures="0" errors="1">
   <testsuite name="AllAccountMatching" tests="1" failures="0" errors="1" time="73" timestamp="2019-05-07 15:27:36" hostname="dipak - DIPAKDAS-PC.indusnettechnologies.com" id="Test Suites/AllAccountMatching">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="E:\\MarkitCore\\CoreApplication\\Reports\\AllAccountMatching\\20190507_152730"/>
         <property name="logFiles" value="E:\\MarkitCore\\CoreApplication\\Reports\\AllAccountMatching\\20190507_152730\\execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="dipak - DIPAKDAS-PC.indusnettechnologies.com"/>
         <property name="os" value="Windows 7 32bit"/>
         <property name="katalonVersion" value="6.0.5.4"/>
         <property name="browser" value="Chrome 74.0.3729.131"/>
         <property name="hostAddress" value="10.0.12.237"/>
         <property name="sessionId" value="59d98e134edefc2c12862312360f0299"/>
         <property name="seleniumVersion" value="3.7.1"/>
         <property name="proxyInformation" value="ProxyInformation{proxyOption=NO_PROXY, proxyServerType=HTTP, password=, proxyServerAddress=, proxyServerPort=0}"/>
         <property name="platform" value="Windows 7"/>
      </properties>
      <testcase name="Test Cases/TC01-LoginApplication" classname="Test Cases/TC01-LoginApplication" status="PASSED">
         <system-out><![CDATA[2019-05-07 15:27:45 - [TEST_CASE][PASSED] - Test Cases/TC01-LoginApplication: Test Cases/TC01-LoginApplication

2019-05-07 15:27:45 - [TEST_STEP][NOT_RUN] - openBrowser(""): NOT_RUN: openBrowser("")

2019-05-07 15:27:45 - [MESSAGE][NOT_RUN] - NOT_RUN: openBrowser("")

2019-05-07 15:27:45 - [TEST_STEP][NOT_RUN] - navigateToUrl("https://dev.offshoresoftwaresolutions.net/markit_coreV2/Sites/CoreTest/login/admin_display"): NOT_RUN: navigateToUrl("https://dev.offshoresoftwaresolutions.net/markit_coreV2/Sites/CoreTest/login/admin_display")

2019-05-07 15:27:45 - [MESSAGE][NOT_RUN] - NOT_RUN: navigateToUrl("https://dev.offshoresoftwaresolutions.net/markit_coreV2/Sites/CoreTest/login/admin_display")

2019-05-07 15:27:45 - [TEST_STEP][PASSED] - setText(findTestObject("Object Repository/TC-LoginApplication/Page_Markit core new  Login/input_Login to your account_username"), username): Text 'dDasTester' is set on object 'Object Repository/TC-LoginApplication/Page_Markit core new  Login/input_Login to your account_username'

2019-05-07 15:27:45 - [MESSAGE][PASSED] - Text 'dDasTester' is set on object 'Object Repository/TC-LoginApplication/Page_Markit core new  Login/input_Login to your account_username'

2019-05-07 15:27:45 - [TEST_STEP][PASSED] - setText(findTestObject("Object Repository/TC-LoginApplication/Page_Markit core new  Login/input_Login to your account_password"), password): Text 'Czybm6R*' is set on object 'Object Repository/TC-LoginApplication/Page_Markit core new  Login/input_Login to your account_password'

2019-05-07 15:27:45 - [MESSAGE][PASSED] - Text 'Czybm6R*' is set on object 'Object Repository/TC-LoginApplication/Page_Markit core new  Login/input_Login to your account_password'

2019-05-07 15:27:45 - [TEST_STEP][PASSED] - delay(4, STOP_ON_FAILURE): Delayed 4 second(s)

2019-05-07 15:27:49 - [MESSAGE][PASSED] - Delayed 4 second(s)

2019-05-07 15:27:49 - [TEST_STEP][NOT_RUN] - maximizeWindow(): NOT_RUN: maximizeWindow()

2019-05-07 15:27:49 - [MESSAGE][NOT_RUN] - NOT_RUN: maximizeWindow()

2019-05-07 15:27:49 - [TEST_STEP][PASSED] - click(findTestObject("Object Repository/TC-LoginApplication/Page_Markit core new  Login/button_Login")): Object: 'Object Repository/TC-LoginApplication/Page_Markit core new  Login/button_Login' is clicked on

2019-05-07 15:27:55 - [MESSAGE][PASSED] - Object: 'Object Repository/TC-LoginApplication/Page_Markit core new  Login/button_Login' is clicked on

2019-05-07 15:27:55 - [TEST_STEP][NOT_RUN] - log = new com.kms.katalon.core.logging.KeywordLogger(): NOT_RUN: log = new com.kms.katalon.core.logging.KeywordLogger()

2019-05-07 15:27:55 - [MESSAGE][NOT_RUN] - NOT_RUN: log = new com.kms.katalon.core.logging.KeywordLogger()

2019-05-07 15:27:55 - [TEST_STEP][NOT_RUN] - log.logInfo(“Dipak”): NOT_RUN: log.logInfo(“Dipak”)

2019-05-07 15:27:55 - [MESSAGE][NOT_RUN] - NOT_RUN: log.logInfo(“Dipak”)]]></system-out>
         <system-err><![CDATA[]]></system-err>
      </testcase>
      <testcase name="Test Cases/TC08-AccountMatching-Insurer" classname="Test Cases/TC08-AccountMatching-Insurer" status="ERROR">
         <error type="ERROR" message="Test Cases/TC08-AccountMatching-Insurer FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC08-AccountMatching-Insurer.run(TC08-AccountMatching-Insurer:79)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557223050455.run(TempTestSuite1557223050455.groovy:35)
Caused by: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1555058886705.run(Script1555058886705.groovy:79)
	... 13 more
"/>
         <system-out><![CDATA[2019-05-07 15:27:55 - [TEST_CASE][ERROR] - Test Cases/TC08-AccountMatching-Insurer: Test Cases/TC08-AccountMatching-Insurer FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC08-AccountMatching-Insurer.run(TC08-AccountMatching-Insurer:79)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557223050455.run(TempTestSuite1557223050455.groovy:35)
Caused by: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1555058886705.run(Script1555058886705.groovy:79)
	... 13 more


2019-05-07 15:27:56 - [TEST_STEP][PASSED] - log = new com.kms.katalon.core.logging.KeywordLogger(): null

2019-05-07 15:27:56 - [TEST_STEP][PASSED] - delay(3): Delayed 3 second(s)

2019-05-07 15:27:59 - [MESSAGE][PASSED] - Delayed 3 second(s)

2019-05-07 15:27:59 - [TEST_STEP][PASSED] - scrollToElement(findTestObject("TC04-AccountMatching-MGA/AccountManagement"), 5): Scroll to object 'Object Repository/TC04-AccountMatching-MGA/AccountManagement' successfully

2019-05-07 15:27:59 - [MESSAGE][PASSED] - Scroll to object 'Object Repository/TC04-AccountMatching-MGA/AccountManagement' successfully

2019-05-07 15:27:59 - [TEST_STEP][PASSED] - click(findTestObject("TC04-AccountMatching-MGA/AccountManagement")): Object: 'Object Repository/TC04-AccountMatching-MGA/AccountManagement' is clicked on

2019-05-07 15:27:59 - [MESSAGE][PASSED] - Object: 'Object Repository/TC04-AccountMatching-MGA/AccountManagement' is clicked on

2019-05-07 15:27:59 - [TEST_STEP][NOT_RUN] - delay(5): NOT_RUN: delay(5)

2019-05-07 15:27:59 - [MESSAGE][NOT_RUN] - NOT_RUN: delay(5)

2019-05-07 15:27:59 - [TEST_STEP][PASSED] - waitForElementVisible(findTestObject("TC04-AccountMatching-MGA/AccountMatchingLink"), 4): Object 'Object Repository/TC04-AccountMatching-MGA/AccountMatchingLink' is visible

2019-05-07 15:27:59 - [MESSAGE][PASSED] - Object 'Object Repository/TC04-AccountMatching-MGA/AccountMatchingLink' is visible

2019-05-07 15:27:59 - [TEST_STEP][PASSED] - click(findTestObject("TC04-AccountMatching-MGA/AccountMatchingLink")): Object: 'Object Repository/TC04-AccountMatching-MGA/AccountMatchingLink' is clicked on

2019-05-07 15:28:01 - [MESSAGE][PASSED] - Object: 'Object Repository/TC04-AccountMatching-MGA/AccountMatchingLink' is clicked on

2019-05-07 15:28:01 - [TEST_STEP][PASSED] - delay(5): Delayed 5 second(s)

2019-05-07 15:28:06 - [MESSAGE][PASSED] - Delayed 5 second(s)

2019-05-07 15:28:06 - [TEST_STEP][PASSED] - selectOptionByLabel(findTestObject("TC04-AccountMatching-MGA/EntityType"), "Insurer", false): All options with label 'Insurer' are selected in object 'Object Repository/TC04-AccountMatching-MGA/EntityType' 

2019-05-07 15:28:06 - [MESSAGE][PASSED] - All options with label 'Insurer' are selected in object 'Object Repository/TC04-AccountMatching-MGA/EntityType' 

2019-05-07 15:28:06 - [TEST_STEP][PASSED] - delay(5): Delayed 5 second(s)

2019-05-07 15:28:11 - [MESSAGE][PASSED] - Delayed 5 second(s)

2019-05-07 15:28:11 - [TEST_STEP][PASSED] - selectOptionByLabel(findTestObject("TC04-AccountMatching-MGA/Entity"), "New Insurer", false): All options with label 'New Insurer' are selected in object 'Object Repository/TC04-AccountMatching-MGA/Entity' 

2019-05-07 15:28:11 - [MESSAGE][PASSED] - All options with label 'New Insurer' are selected in object 'Object Repository/TC04-AccountMatching-MGA/Entity' 

2019-05-07 15:28:11 - [TEST_STEP][PASSED] - waitForElementClickable(findTestObject("TC04-AccountMatching-MGA/ClickGoButton"), 5): Object 'Object Repository/TC04-AccountMatching-MGA/ClickGoButton' is clickable

2019-05-07 15:28:11 - [MESSAGE][PASSED] - Object 'Object Repository/TC04-AccountMatching-MGA/ClickGoButton' is clickable

2019-05-07 15:28:11 - [TEST_STEP][PASSED] - click(findTestObject("TC04-AccountMatching-MGA/ClickGoButton")): Object: 'Object Repository/TC04-AccountMatching-MGA/ClickGoButton' is clicked on

2019-05-07 15:28:13 - [MESSAGE][PASSED] - Object: 'Object Repository/TC04-AccountMatching-MGA/ClickGoButton' is clicked on

2019-05-07 15:28:13 - [TEST_STEP][PASSED] - waitForJQueryLoad(5): jQuery is ready.

2019-05-07 15:28:14 - [MESSAGE][PASSED] - jQuery is ready.

2019-05-07 15:28:14 - [TEST_STEP][PASSED] - selectOptionByLabel(findTestObject("TC04-AccountMatching-MGA/SelectAccount"), "New Insurer Comission Account -- Allocated -- New Insurer-COM-ACC -- USD", false): All options with label 'New Insurer Comission Account -- Allocated -- New Insurer-COM-ACC -- USD' are selected in object 'Object Repository/TC04-AccountMatching-MGA/SelectAccount' 

2019-05-07 15:28:14 - [MESSAGE][PASSED] - All options with label 'New Insurer Comission Account -- Allocated -- New Insurer-COM-ACC -- USD' are selected in object 'Object Repository/TC04-AccountMatching-MGA/SelectAccount' 

2019-05-07 15:28:14 - [TEST_STEP][PASSED] - delay(5): Delayed 5 second(s)

2019-05-07 15:28:19 - [MESSAGE][PASSED] - Delayed 5 second(s)

2019-05-07 15:28:19 - [TEST_STEP][PASSED] - setText(findTestObject("TC04-AccountMatching-MGA/Acc_Matching_PolicyNo_TextBox"), MTA_Policy_No): Text 'DASD/POLICY/003057' is set on object 'Object Repository/TC04-AccountMatching-MGA/Acc_Matching_PolicyNo_TextBox'

2019-05-07 15:28:19 - [MESSAGE][PASSED] - Text 'DASD/POLICY/003057' is set on object 'Object Repository/TC04-AccountMatching-MGA/Acc_Matching_PolicyNo_TextBox'

2019-05-07 15:28:19 - [TEST_STEP][PASSED] - sendKeys(findTestObject("TC04-AccountMatching-MGA/Acc_Matching_PolicyNo_TextBox"), Keys.chord(ENTER)): Keys 'ENTER + ' sent to object: 'Object Repository/TC04-AccountMatching-MGA/Acc_Matching_PolicyNo_TextBox'

2019-05-07 15:28:19 - [MESSAGE][PASSED] - Keys 'ENTER + ' sent to object: 'Object Repository/TC04-AccountMatching-MGA/Acc_Matching_PolicyNo_TextBox'

2019-05-07 15:28:19 - [TEST_STEP][PASSED] - delay(5): Delayed 5 second(s)

2019-05-07 15:28:24 - [MESSAGE][PASSED] - Delayed 5 second(s)

2019-05-07 15:28:24 - [TEST_STEP][PASSED] - waitForElementClickable(findTestObject("TC04-AccountMatching-MGA/CheckBox"), 5): Web element with id: 'Object Repository/TC04-AccountMatching-MGA/CheckBox' located by 'By.xpath: //table//input[@class='TrChkClass']' not found

2019-05-07 15:28:29 - [MESSAGE][INFO] - Unable to find the element located by 'By.xpath: //table//input[@class='TrChkClass']'. Please recheck the objects properties to make sure the desired element is located. 

2019-05-07 15:28:29 - [MESSAGE][WARNING] - Web element with id: 'Object Repository/TC04-AccountMatching-MGA/CheckBox' located by 'By.xpath: //table//input[@class='TrChkClass']' not found

2019-05-07 15:28:29 - [TEST_STEP][PASSED] - click(findTestObject("TC04-AccountMatching-MGA/CheckBox")): Object: 'Object Repository/TC04-AccountMatching-MGA/CheckBox' is clicked on

2019-05-07 15:28:34 - [MESSAGE][PASSED] - Object: 'Object Repository/TC04-AccountMatching-MGA/CheckBox' is clicked on

2019-05-07 15:28:34 - [TEST_STEP][PASSED] - delay(4): Delayed 4 second(s)

2019-05-07 15:28:38 - [MESSAGE][PASSED] - Delayed 4 second(s)

2019-05-07 15:28:38 - [TEST_STEP][PASSED] - Insurer_Comission_Amount = getText(findTestObject("TC04-AccountMatching-MGA/MGA_Amount")): Text of object 'Object Repository/TC04-AccountMatching-MGA/MGA_Amount' is: 'USD 84.00'

2019-05-07 15:28:38 - [MESSAGE][PASSED] - Text of object 'Object Repository/TC04-AccountMatching-MGA/MGA_Amount' is: 'USD 84.00'

2019-05-07 15:28:38 - [TEST_STEP][PASSED] - Insur_Comm_Amt = subString.FatchSubstringValue.SubString(Insurer_Comission_Amount): subString.FatchSubstringValue.SubString is PASSED

2019-05-07 15:28:38 - [MESSAGE][PASSED] - subString.FatchSubstringValue.SubString is PASSED

2019-05-07 15:28:38 - [TEST_STEP][NOT_RUN] - scrollToElement(findTestObject("TC04-AccountMatching-MGA/PAYFULL"), 5): NOT_RUN: scrollToElement(findTestObject("TC04-AccountMatching-MGA/PAYFULL"), 5)

2019-05-07 15:28:38 - [MESSAGE][NOT_RUN] - NOT_RUN: scrollToElement(findTestObject("TC04-AccountMatching-MGA/PAYFULL"), 5)

2019-05-07 15:28:38 - [TEST_STEP][NOT_RUN] - setText(findTestObject("TC04-AccountMatching-MGA/PAYFULL"), Insurer_Comission_Amount): NOT_RUN: setText(findTestObject("TC04-AccountMatching-MGA/PAYFULL"), Insurer_Comission_Amount)

2019-05-07 15:28:38 - [MESSAGE][NOT_RUN] - NOT_RUN: setText(findTestObject("TC04-AccountMatching-MGA/PAYFULL"), Insurer_Comission_Amount)

2019-05-07 15:28:38 - [TEST_STEP][PASSED] - click(findTestObject("TC04-AccountMatching-MGA/PayFullClick")): Object: 'Object Repository/TC04-AccountMatching-MGA/PayFullClick' is clicked on

2019-05-07 15:28:38 - [MESSAGE][PASSED] - Object: 'Object Repository/TC04-AccountMatching-MGA/PayFullClick' is clicked on

2019-05-07 15:28:38 - [TEST_STEP][PASSED] - delay(4): Delayed 4 second(s)

2019-05-07 15:28:42 - [MESSAGE][PASSED] - Delayed 4 second(s)

2019-05-07 15:28:42 - [TEST_STEP][PASSED] - click(findTestObject("TC04-AccountMatching-MGA/MatchAmount")): Object: 'Object Repository/TC04-AccountMatching-MGA/MatchAmount' is clicked on

2019-05-07 15:28:42 - [MESSAGE][PASSED] - Object: 'Object Repository/TC04-AccountMatching-MGA/MatchAmount' is clicked on

2019-05-07 15:28:42 - [TEST_STEP][PASSED] - delay(5): Delayed 5 second(s)

2019-05-07 15:28:47 - [MESSAGE][PASSED] - Delayed 5 second(s)

2019-05-07 15:28:47 - [TEST_STEP][ERROR] - com.excel.helper.ExcelHelper.updateTheExcel("E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx", "Insurer", Insur_Comm_Amt): com.excel.helper.ExcelHelper.updateTheExcel("E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx", "Insurer", Insur_Comm_Amt) FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC08-AccountMatching-Insurer.run(TC08-AccountMatching-Insurer:79)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557223050455.run(TempTestSuite1557223050455.groovy:35)
Caused by: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1555058886705.run(Script1555058886705.groovy:79)
	... 13 more


2019-05-07 15:28:49 - [MESSAGE][ERROR] - java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)

2019-05-07 15:28:49 - [MESSAGE][ERROR] - com.excel.helper.ExcelHelper.updateTheExcel("E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx", "Insurer", Insur_Comm_Amt) FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC08-AccountMatching-Insurer.run(TC08-AccountMatching-Insurer:79)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557223050455.run(TempTestSuite1557223050455.groovy:35)
Caused by: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1555058886705.run(Script1555058886705.groovy:79)
	... 13 more]]></system-out>
         <system-err><![CDATA[2019-05-07 15:27:55 - [TEST_CASE][ERROR] - Test Cases/TC08-AccountMatching-Insurer: Test Cases/TC08-AccountMatching-Insurer FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC08-AccountMatching-Insurer.run(TC08-AccountMatching-Insurer:79)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557223050455.run(TempTestSuite1557223050455.groovy:35)
Caused by: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1555058886705.run(Script1555058886705.groovy:79)
	... 13 more]]></system-err>
      </testcase>
      <system-out><![CDATA[2019-05-07 15:27:36 - [TEST_SUITE][ERROR] - AllAccountMatching: Test Cases/TC08-AccountMatching-Insurer FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC08-AccountMatching-Insurer.run(TC08-AccountMatching-Insurer:79)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557223050455.run(TempTestSuite1557223050455.groovy:35)
Caused by: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1555058886705.run(Script1555058886705.groovy:79)
	... 13 more]]></system-out>
      <system-err><![CDATA[2019-05-07 15:27:36 - [TEST_SUITE][ERROR] - AllAccountMatching: Test Cases/TC08-AccountMatching-Insurer FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC08-AccountMatching-Insurer.run(TC08-AccountMatching-Insurer:79)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:331)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:322)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:293)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:227)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1557223050455.run(TempTestSuite1557223050455.groovy:35)
Caused by: java.io.FileNotFoundException: E:\MarkitCore\CoreApplication\Data Files\TestData\Comission.xlsx (The process cannot access the file because it is being used by another process)
	at com.excel.helper.ExcelHelper.writeToExcel(Excelhelper.groovy:78)
	at com.excel.helper.ExcelHelper.updateTheExcel(Excelhelper.groovy:62)
	at com.excel.helper.ExcelHelper.invokeMethod(Excelhelper.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at Script1555058886705.run(Script1555058886705.groovy:79)
	... 13 more]]></system-err>
   </testsuite>
</testsuites>
